File: src/test/java/net/rentalhost/plugins/php/hammer/inspections/codeStyle/NullableTypesFormatInspectionTestCase.java
package net.rentalhost.plugins.php.hammer.inspections.codeStyle;

import net.rentalhost.plugins.services.TestCase;

public class NullableTypesFormatInspectionTestCase
    extends TestCase {
    public void testAll() {
        testInspection(NullableTypesFormatInspection.class);
    }
}

File: src/test/java/net/rentalhost/plugins/php/hammer/inspections/codeStyle/NullableTypesFormatInspectionTestCase.kt
package net.rentalhost.plugins.php.hammer.inspections.codeStyle

import net.rentalhost.plugins.services.TestCase

class NullableTypesFormatInspectionTestCase : TestCase() {
    fun testAll() = testInspection(NullableTypesFormatInspection::class.java)
}
